From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

4c74356b41 commented on articles/virtual-machines/virtual-machines-windows-migration-classic-resource-manager.md:
```diff
@@ -19,11 +19,11 @@
 ## What is our goal with migration?
 
-We are really excited about the power and capability offered by the new experience and APIs available on Virtual Machines. We think this is really going to change the way you can use the cloud in a lot of ways. with the release of the new model, it allows you to deploy, manage, and monitor related services in a resource group. Resource Manager enables deploying complex applications using template ...
+We are really excited about the power and capability offered by the new experience and APIs available on Virtual Machines. We think this is really going to change the way you can use the cloud in a lot of ways. With the release of the new model, it allows you to deploy, manage, and monitor related services in a resource group. Resource Manager enables deploying complex applications using template ...
```
virtual machine to Virtual Machine

4c74356b41 commented on articles/virtual-machines/virtual-machines-windows-migration-classic-resource-manager.md:
```diff
@@ -40,29 +40,29 @@ Please note that one of the important changes that you will have to make as part
 
 - Management Plane - This describes the calls that comes into the management plane or the API for modifying resources. For example – Creating a VM, Restarting a VM, Update a Virtual Network with a new subnet, etc. All of these operations manage the resources running but doesn't directly impact connecting to the instances.
-- Data Plane (Application) – This describes the “runtime” of the application itself and involve interaction with instances that don’t go through the Azure API. Accessing your website or pulling data from a running SQL Server or mongoDB server would all be considered data plane or application interaction. Copying a blob from a storage account and accessing a Public IP address to RDP or SSH into th ...
+- Data Plane (Application) – This describes the “runtime” of the application itself and involve interaction with instances that don’t go through the Azure API. Accessing your website or pulling data from a running SQL Server or MongoDB server would all be considered data plane or application interaction. Copying a blob from a storage account and accessing a Public IP address to RDP or SSH into th ...
```
mongoDB to MongoDB

tysonn commented:
@mahthi - Please review this public repo PR from @4c74356b41

mahthi commented:
Looks good to me! Thanks!

4c74356b41 commented:
Can you please look into this?\

It’s been almost a year since we announced the support for Virtual Machines under Azure Resource Manager. You can read more about the advancements and the additional capabilities that it supports >>>here<<< no link. since I'm not sure what link do you want i'm not inserting any
And there’s a focus of enabling security by default with the enforcement of virtual machines in a Virtual Network. - shouldn't it be "focus ON"?

mahthi commented:
It’s been almost a year since we [announced support for Virtual Machines under Azure Resource Manager.](http://azure.microsoft.com/blog/2015/04/29/iaas-just-got-easier-again/) You can read more about the [advancements ](http://azure.microsoft.com/en-us/documentation/articles/virtual-machines-azurerm-versus-azuresm/)and the [additional capabilities](http://azure.microsoft.com/blog/2015/07/30/tagging-on-compute-network-and-storage-resources/) of the new stack. We also gave [guidance ](http://azure.microsoft.com/en-us/documentation/articles/virtual-networks-arm-asm-s2s-howto/)around how to best connect and have resources from the two deployment models co-exist in your subscription using Virtual Network Site-Site Gateways. In addition, we released the capability to [configure your ExpressRoute circuit to connect to both your classic and resource manager virtual networks](http://azure.microsoft.com/en-us/documentation/articles/expressroute-move/).

You can update it with the above text to make it cleaner.

4c74356b41 commented:
I'm sorry, I'm kinda new to this, how do I incorporate that into my PR and keep links working?

tysonn commented:
@mahthi - I'd recommend you submit the update in a pull request against @4c74356b41 fork/branch. Then, all @4c74356b41 has to do is merge your pull request.

4c74356b41 commented:
this stuff now says that it has conflicts that must be resolved =\

4c74356b41 commented:
Well, are you folks actually going to do anything about those 2 fixes or not?

mahthi commented:
Its in my backlog and I'll get to it next week.

tysonn commented:
I have taken care of this. @4c74356b41, thanks for the updates.

GitHubber17 committed:
Merge pull request #35925 from dlepow/vmzone

VM zone: portal screen

ShawnJackson committed:
Update log-analytics-oms-gateway.md

Commit:
Merge branch 'release-availabilityzones-ga' of https://github.com/MicrosoftDocs/azure-docs-pr into zr-images

Commit:
minor updates

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

PRMerger10 committed:
Merge pull request #35868 from cynthn/zr-images

Image Zone resilient stuffs

Commit:
Merge branch 'release-availabilityzones-ga' of https://github.com/MicrosoftDocs/azure-docs-pr into march-snap

Commit:
fixing formatting

gatneil committed:
some updates

mmacy committed:
[ACI] new article: exec

mmacy committed:
[ACI] minor copy mod

mmacy committed:
Merge branch 'release-ga-aci' of https://github.com/MicrosoftDocs/azure-docs-pr into aci-exec

rjagiewich committed:
Merge pull request #35867 from cynthn/march-snap

Snapshots ZRS additions

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

iainfoulds committed:
Update az-overview.md

iainfoulds committed:
[Availability Zone] Removing locale from link

Lisaco88 committed:
Merge pull request #35978 from iainfoulds/patch-21

[Availability Zones] Updating with GA content

mmacy committed:
[ACI] one-liners

mmacy committed:
Merge branch 'release-ga-aci' of https://github.com/MicrosoftDocs/azure-docs-pr into aci-exec

KumudD committed:
tweak

KumudD committed:
fixed merge conflicts

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

KumudD committed:
Merge branch 'release-availabilityzones-ga' of https://github.com/Microsoft/azure-docs-pr into release-availabilityzones-ga

KumudD committed:
tweak

mmacy committed:
[ACI] more exec particulars

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

KumudD committed:
updated content

KumudD committed:
Merge branch 'release-availabilityzones-ga' of https://github.com/Microsoft/azure-docs-pr into release-availabilityzones-ga

KumudD committed:
addressed PR Reviewer's feedback

KumudD committed:
adddressed feedback

GitHubber17 committed:
Merge pull request #35025 from KumudD/release-availabilityzones-ga

Standard Load Balancer & AZ

KumudD committed:
new topic added

KumudD committed:
title update

KumudD committed:
updated TOC

KumudD committed:
fixed formatting

KumudD committed:
tweak

KumudD committed:
new files

KumudD committed:
feedback incorp

mmacy committed:
[ACI] add ".exe" for more Windowsness

KumudD committed:
toc update

KumudD committed:
fixed links

KumudD committed:
toc update

KumudD committed:
updated toc

KumudD committed:
fixed link

KumudD committed:
fixed links

KumudD committed:
fixed image

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

rmca14 committed:
Merge pull request #36218 from KumudD/release-availabilityzones-ga

Load balance VMs across zones -portal

rjagiewich committed:
Update log-analytics-oms-gateway.md

KumudD committed:
fixed merge conflicts

KumudD committed:
renamed files

KumudD committed:
fixed broken links

Commit:
Removed preview info.

ktoliver committed:
Merge pull request #36233 from KumudD/zonallbazga

Added Zonal LB topics

tamram committed:
Merge branch 'release-availabilityzones-ga' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-az

PRMerger16 committed:
Merge pull request #36292 from jimdial/release-availabilityzones-ga

Removed preview info.

tamram committed:
updating metadata for includes

tamram committed:
fixing up links

ktoliver committed:
Merge pull request #35926 from tamram/tamram-az

storage updates for AZ GA

ktoliver committed:
Merge pull request #36065 from mmacy/aci-exec

[ACI] new article: exec

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

tolandmike committed:
updating table - replication

tolandmike committed:
test

liwong88 committed:
moved how-to section

gatneil committed:
added clarification

tolandmike committed:
updated the table

v-alje committed:
Bringing even with master.

mmacy committed:
[ACI] exec ref link fix

Commit:
Merge pull request #36409 from mmacy/aci-exec-link-fix

[ACI] exec ref link fix

tamram committed:
adding ZRS classic article to TOCs

GitHubber17 committed:
Merge pull request #36429 from tamram/release-availabilityzones-ga

adding ZRS classic article to TOCs

tolandmike committed:
updated ZRS

v-alje committed:
Resolving merge conflict

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

Commit:
ca05

tolandmike committed:
regions, and merged ZRS Classic

v-alje committed:
Bringing even with master.

Commit:
Migrate CLI 1.0 install directions to the CLI repo

v-alje committed:
Resolving merge conflicts

v-alje committed:
Bringing even with master.

liwong88 committed:
re-arranged TOC & added breadcrumb

v-alje committed:
Bringing even with master.

liwong88 committed:
rearraged TOC

littleaj committed:
Update app-insights-java-get-started.md

added information about configuration file directory.
some adjustments to spacing, formatting, grammar, consistency.

littleaj committed:
Update app-insights-java-get-started.md

updates based on acrolinx scorecard suggestions

tolandmike committed:
updates with Tamra's and Dinesh's feedback

tamram committed:
removing ZRS classic from TOC

tolandmike committed:
Merge pull request #1 from tamram/tamram-36454

removing ZRS classic from TOC

alkohli committed:
added support doc

tamram committed:
Merge branch 'release-availabilityzones-ga' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-36454

tamram committed:
Merge branch 'jeffking-0326' of https://github.com/tolandmike/azure-docs-pr into tamram-az-0329

alkohli committed:
Fixed broken links

tamram committed:
removing ZRS links from TOCs

rjagiewich committed:
Update log-analytics-oms-gateway.md

updating links per author's request

alkohli committed:
modified TOC

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

alkohli committed:
more changes

alkohli committed:
acrolinx changes

Commit:
Update author field in existing articles

MGoedtel committed:
updated limit article due to LA pricing changes

KumudD committed:
removed preview text

cherylmc committed:
format

Commit:
Merge pull request #36626 from KumudD/removepreviewtextaz

removed preview text

cherylmc committed:
graphics

cherylmc committed:
format

Commit:
changed the location of config settings file

Commit:
Connection-monitor-GA

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nw-connection-monitor-ga

mimig1 committed:
Cosmos DB: Added links to new/updated videos

rjagiewich committed:
Update log-analytics-oms-gateway.md

mimig1 committed:
Acrolinx fix

mimig1 committed:
fixed one link

Commit:
Fix

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nw-connection-monitor-ga

tfitzmac committed:
updated move for ML

mimig1 committed:
link fix

mimig1 committed:
link fix

tfitzmac committed:
clarification

billmath committed:
adding tou march update

v-geberr committed:
fix images for new UX

rboucher committed:
update monitor-alerts-unified to remove preview and reword. Also TOC is not publishing

bandersmsft committed:
azure policy - setting nini to reviewer

tamram committed:
incorp feedback

rboucher committed:
remove spaces

alkohli committed:
changes from PR reviewers

v-geberr committed:
fix key string copy

rboucher committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fixalerts2

v-geberr committed:
new screen shots

Ja-Dunn committed:
Update Label-Suggested-Utterances.md

capping

rboucher committed:
change the note in insights-alert-portal

rboucher committed:
fix numbering problem 8.a

Commit:
Merge pull request #36646 from tfitzmac/0329move

updated move for ML

Commit:
reverting files

Commit:
Merge pull request #36650 from bandersmsft/master

azure policy - setting nini to reviewer

Ja-Dunn committed:
Merge pull request #36328 from lisawong19/cognitive-comp-vision

moved how-to section & changed title of overview page

Ja-Dunn committed:
Merge pull request #36649 from v-geberr/0329-screenshot-fix-5

fix images for new UX

Ja-Dunn committed:
Merge pull request #36585 from alkohli/support

added support doc

MGoedtel committed:
updated limit article

Ja-Dunn committed:
Merge pull request #36647 from billmath/tou

adding tou march update

Ja-Dunn committed:
Merge pull request #36652 from v-geberr/0329-fix-images-06-maybe

fix key string copy

Ja-Dunn committed:
Merge pull request #36656 from v-geberr/0329-fix-images-07

new screen shots

Commit:
change documentdb to cosmos-db

ktoliver committed:
Merge pull request #36651 from rboucher/fixalerts2

update monitor-alerts-unified to remove preview and reword. Also TOC …

ggailey777 committed:
Set up experiment to test moving the Azure account requirement down in the topic

rboucher committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fixalerts3

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#6492

In the sentence "You must create and log on as a second administrator account to join Azure AD mannually" the word mannually should be spelt manually.

MGoedtel committed:
updated note

Ja-Dunn committed:
Merge pull request #36660 from craigcaseyMSFT/v-craic0329

change documentdb to cosmos-db

gatneil committed:
merged MVC changes

rboucher committed:
change OMS and extend to OMS portal and copy. add classic to older alerts

PRMerger10 committed:
Merge pull request #36662 from MicahMcKittrick-MSFT/patch-11

Fixes MicrosoftDocs/azure-docs#6492

v-alje committed:
Bringing even with master.

ShawnJackson committed:
Update log-analytics-oms-gateway.md

gatneil committed:
fixed link

vladvino committed:
Referenced OAS import blog

v-geberr committed:
manage versions

Ja-Dunn committed:
Merge pull request #36542 from lisawong19/cog-emotion

re-arranged TOC & added breadcrumb

Ja-Dunn committed:
Merge pull request #36549 from lisawong19/cog-face

re-arranged TOC

v-alje committed:
Bringing even with master.

ShawnJackson committed:
Merge pull request #34952 from rjagiewich/log-analytics-oms-gateway

edit pass: Log analytics oms gateway

MGoedtel committed:
added metadata

gatneil committed:
fixed link

PRMerger18 committed:
Merge pull request #36665 from vladvino/master

Referenced OAS import blog

PRMerger19 committed:
Merge pull request #36453 from MarkusVi/ca05

ca05

Commit:
Merge pull request #36329 from gatneil/zonesUpdates

Zones updates

MGoedtel committed:
updated TOC

PRMerger12 committed:
Merge pull request #36556 from littleaj/patch-4

Add config file directory property instructions; minor adjustments

rboucher committed:
numerous updates to the alerts overview articles to consolidate and correct them

rboucher committed:
fix broken link

tamram committed:
fix acrolinx score

sanjeev3 committed:
Update sdk-and-samples.md

ktoliver committed:
Merge pull request #36625 from MGoedtel/UpdateSubLimit329

updated limit article due to LA pricing changes

Commit:
Merge pull request #36679 from sanjeev3/master

Update sdk-and-samples.md

ktoliver committed:
Merge pull request #36628 from cherylmc/change3

format

ktoliver committed:
Merge pull request #36673 from MGoedtel/UpdateOMTOC329

updated TOC

BryanTrach-MSFT committed:
fixes MicrosoftDocs/azure-docs#6334

Updated the PowerShell links that were causing 404 errors. Also saw that CLI 2.0 examples were being given but CLI 1.0 docs were being linked. Updated this as well.

v-geberr committed:
edits

ktoliver committed:
Merge pull request #36587 from tamram/tamram-az-0329

Storage updates for AZ GA

PRMerger16 committed:
Merge pull request #36680 from BryanTrach-MSFT/patch-10

fixes MicrosoftDocs/azure-docs#6334

ktoliver committed:
Merge pull request #36666 from v-geberr/0329-fix-images-08

manage versions

aliuy committed:
Cosmos DB Data Migration Tool

Add link to updated binary

christiankuhtz committed:
Preview removal

christiankuhtz committed:
Removed preview

rolyon committed:
Refreshed descriptions for built-in roles

rolyon committed:
Fixed Virtual Machine Contributor descriptions

v-geberr committed:
new ddl in ux

v-geberr committed:
red box

v-geberr committed:
removed personally identifying info

v-geberr committed:
new screen shot

PRMerger12 committed:
Merge pull request #36686 from christiankuhtz/patch-114

Removed preview

PRMerger13 committed:
Merge pull request #36685 from christiankuhtz/patch-113

Preview removal

v-geberr committed:
new ux screenshots

v-geberr committed:
cities instead of meat

MGoedtel committed:
fixed formatting issue for step 5

mimig1 committed:
Updated Andrew's videos

PRMerger16 committed:
Merge pull request #36681 from aliuy/master

Cosmos DB Data Migration Tool

mimig1 committed:
Cosmos DB: Typo fix.

mattbriggs committed:
Add note to the replace disk topic

itechedit committed:
Merge pull request #36635 from jimdial/nw-connection-monitor-ga

NW connection monitor GA

mimig1 committed:
Small updates

mattbriggs committed:
remove note

Commit:
Merge pull request #36703 from mimig1/patch-235

Cosmos DB: Typo fix.

mattbriggs committed:
update note

rmca14 committed:
Merge pull request #36658 from cynthn/remove-zones

Reverting files

itechedit committed:
Merge pull request #36663 from rboucher/fixalerts3

update various text and terms for release

rmca14 committed:
Merge pull request #36661 from ggailey777/experiment

Set up experiment to test moving the Azure account requirement later

mimig1 committed:
Cosmos DB: Updated calculator screenshots

GitHubber17 committed:
Merge pull request #36688 from rolyon/rolyon-issue6250

[Azure AD RBAC] Update descriptions for built-in roles for issue 6250

v-alje committed:
Bringing even with master.

PRMerger14 committed:
Merge pull request #36642 from mimig1/03292018videos

Cosmos DB: Added links to new/updated videos

GitHubber17 committed:
Merge pull request #36695 from v-geberr/0331-screenshot-fix

0331 screenshot fix

GitHubber17 committed:
Merge pull request #36696 from v-geberr/0330-screenshot-fix-3

new screen shot

GitHubber17 committed:
Merge pull request #36697 from v-geberr/0330-screenshot-fix-2

removed personally identifying info

GitHubber17 committed:
Merge pull request #36701 from v-geberr/0330-screenshot-fix-05

0330 screenshot fix 05

v-geberr committed:
new ux (new ddl , remove fuzzy search)

v-alje committed:
Bringing even with master.

Commit:
Merge pull request #36705 from mattbriggs/mdb_20170330_tr_update

Add note to the replace disk topic

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#6483

The table for DS_V2 was not formatted correctly. Corrected this.

Commit:
Merge pull request #36707 from mattbriggs/mdb_20180330_power_removenote

remove note

Commit:
Added qualifier on step 3 of Enable flow logs.

mimig1 committed:
Cosmos DB: Updated firewall support screenshots

Commit:
Added qualifier to enable NSG flow logs.

Commit:
Added qualifier to Enable NSG flow logging.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger12 committed:
Merge pull request #36711 from MicahMcKittrick-MSFT/patch-12

Fixes MicrosoftDocs/azure-docs#6483

Commit:
Update

itechedit committed:
Merge pull request #36709 from v-geberr/0330-screenshot-fix-6

new ux (new ddl , remove fuzzy search)

v-alje committed:
Merge pull request #36235 from MicrosoftDocs/release-availabilityzones-ga

Release availabilityzones ga

v-alje committed:
Merge pull request #36654 from MicrosoftDocs/release-ga-aci

Release ga aci

Commit:
Updates.

PRMerger15 committed:
Merge pull request #36716 from jimdial/patch-122

Added qualifier to enable NSG flow logs.

itechedit committed:
Merge pull request #36713 from mimig1/03302018publicpr

Cosmos DB: Updated firewall support screenshots

v-alje committed:
Merge pull request #36718 from MicrosoftDocs/FromPublicRepo

From public repo

GitHubber17 committed:
Merge pull request #36631 from v-payc/ChangingConfigFileLocation

changed the location of config settings file

Commit:
Updates.

itechedit committed:
Merge pull request #36539 from sptramer/cli/migrate-1.0-install

Migrate CLI 1.0 install directions to the CLI repo

MGoedtel committed:
fixed formatting issue for step 5

GitHubber17 committed:
Merge pull request #36714 from mimig1/03302018rus

Cosmos DB: Updated calculator screenshots

itechedit committed:
Merge pull request #36717 from jimdial/patch-123

Added qualifier to Enable NSG flow logging.

Commit:
Merge pull request #36702 from MGoedtel/UpdateSvcFabricLA330

fixed formatting issue for step 5

Commit:
Merge pull request #36712 from jimdial/patch-121

Added qualifier on step 3 of Enable flow logs.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
